doc/ug/getting_started_dv.md

```diff
@@ -61,28 +60,28 @@ There is support for Cadence Xcelium as well, which is being slowly ramped up.
 
 ## Building and Running Tests
 
-The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<ip>_sanity_vseq.sv` for developing the sanity test.
-The sanity test can be run as-is by invoking `make`, as a "hello world" step to bring the DUT out of reset.
+The `uvmdvgen` tool provides an empty shell sequence at `dv/env/seq_lib/<dut>_sanity_vseq.sv` for developing the sanity test.
+The sanity test can be run as-is by invoking `dvsim.py`, as a "hello world" step to bring the DUT out of reset.
 ```console
 $ cd path-to-dv
-$ make [SIMULATOR=xcelium] [WAVES=1]
+$ util/dvsim.py <dut>_sim_cfg.hjson -i <dut>_sanity [-w] [-t xcelium]
 ```
 
 The generated initial testbench is not expected to compile and elaborate successfully right away.
 There may be additional fixes required, which can be hopefully be identified easily.
 Once the testbench compiles and elaborates without any errors or warnings, the sanity sequence can be developed further to access a major datapath and test the basic functionality of the DUT.
 
 VCS is used as the default simulator.
-It can be switched to Xcelium by setting `SIMULATOR=xcelium` on the command line.
-The `WAVES=1` switch will cause an `fsdb` dump to be created from the test.
+It can be switched to Xcelium by setting `-t xcelium` on the command line.
+The `-w` switch will cause an `fsdb` dump to be created from the test.
```

Comment:
As well as expanding this, I'd be less explicit about the wave dumping format (since it depends on the simulator and on whether you have Verdi installed). Something like this maybe? "To dump waves from the simulation, pass the `--waves` argument to `dvsim.py`."

Reply:
I agree, yeah that is a good point, I will update that.